athena-snowflake/assembly.xml

```diff
@@ -0,0 +1,30 @@
+<assembly xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2 http://maven.apache.org/xsd/assembly-1.1.2.xsd">
+    <id>zip</id>
+    <!-- Make sure the ZIP contents are not nested in a subdirectory -->
+    <includeBaseDirectory>false</includeBaseDirectory>
+
+    <formats>
+        <format>zip</format>
+    </formats>
+    <fileSets>
+        <fileSet>
+            <directory>${project.basedir}/conf</directory>
+        </fileSet>
+        <!-- Include the compiled classes as-is and put them in the root of the ZIP -->
+        <fileSet>
+            <directory>${project.build.directory}/classes</directory>
+            <outputDirectory>/</outputDirectory>
+        </fileSet>
+    </fileSets>
+    <dependencySets>
+        <!-- Include all dependencies in the lib/ directory -->
+        <dependencySet>
+            <outputDirectory>lib</outputDirectory>
+            <excludes>
+                <exclude>${project.groupId}:${project.artifactId}:jar:*</exclude>
```

Comment:
please add this to the list of excludes: 

                <excludes>
                    <exclude>**/*.dylib</exclude>
                    <exclude>**/*.dll</exclude>
                    <exclude>META-INF/*.SF</exclude>
                    <exclude>META-INF/*.DSA</exclude>
                    <exclude>META-INF/*.RSA</exclude>
                    <exclude>**/Log4j2Plugins.dat</exclude>
                </excludes>

Reply:
Hi @AbdulR3hman, Using assembly plugin, we are using the format as zip and same is reflected in codeuri.
CodeUri: "./target/athena-snowflake.zip"

regarding exclusions, snowflake connector does not have these dependencies, so i have not excluded it.